.github/workflows/ci.yml

```diff
@@ -15,6 +15,21 @@ env:
 jobs:
   test:
     runs-on: ubuntu-latest
+    services:
+      postgres: # we need this postgres instance for running a local network with indexer and graphql
+        image: postgres
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgrespw
+          POSTGRES_DB: sui_indexer_v2
+          POSTGRES_HOST_AUTH_METHOD: trust
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
```

Comment:
Can we move/factor this out into a separate job that is specific for network tests instead of having it a part of this job?

Reply:
Yes, sounds good!